packages/bot-web-ui/src/components/dashboard/bot-builder/toolbar/__tests__/toolbar.spec.tsx

```diff
@@ -2,14 +2,35 @@ import React from 'react';
 import { render, screen } from '@testing-library/react';
 import Toolbar from '..';
 import { isDesktop, isMobile } from '@deriv/shared';
+import { useDBotStore } from 'Stores/useDBotStore';
 
-jest.mock('Stores/connect', () => ({
-    __esModule: true,
-    default: 'mockedDefaultExport',
-    connect:
-        () =>
-        <T,>(Component: T) =>
-            Component,
+const mockDbotStore: any = {
+    run_panel: {
+        is_running: false,
+    },
+    save_modal: {
+        toggleSaveModal: jest.fn(),
+    },
+    load_modal: {
+        toggleLoadModal: jest.fn(),
+    },
+    toolbar: {
+        has_redo_stack: false,
+        has_undo_stack: false,
+        closeResetDialog: jest.fn(),
+        onResetOkButtonClick: jest.fn(),
+        onResetClick: jest.fn(),
+        onSortClick: jest.fn(),
+        onUndoClick: jest.fn(),
+        onZoomInOutClick: jest.fn(),
+        is_dialog_open: false,
+    },
+    quick_strategy: {},
+    dashboard: {},
+};
+
+jest.mock('Stores/useDBotStore', () => ({
+    useDBotStore: jest.fn(() => mockDbotStore),
```

Comment:
can we use the `mockStore` function here ?

Reply:
we need to do some refactoring in bot-web-ui package to use mockStore or else our test case will fail. We will be making those changes once we start adding test cases in bot-web-ui.